Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,44 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses the interface defined `pwd.h` to add a new user to the system
+password file. 
```

Comment:
This implementation detail is hardly the most important thing about describing `useradd` :)

We can just say that it adds a new user to the system.